src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -34,10 +37,16 @@
 public class OverseasEntitiesController {
 
     private final OverseasEntitiesService overseasEntitiesService;
+    private final OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator;
+
+    @Value("${FEATURE_FLAG_ENABLE_VALIDATION_25082022}")
```

Comment:
Just curious, have we created a corresponding story on the backlog to remove this feature flag? If not, this needs to be done (at the point the feature flag is introduced).

Reply:
Thanks Martin, I will create a story